client/src/containers/Settings/Settings.tsx

```diff
@@ -1,4 +1,5 @@
 import React, { Component, ChangeEvent } from 'react'
+import classnames from 'classnames'
```

Comment:
Do we need a lib to concat 2 values ? Seems like overkill although the syntax is more elegant.

Reply:
Yep.